src/utils/gmime/gmime-filter-html-bq.c

```diff
@@ -219,9 +219,8 @@ citation_cut (char *in, const char *inend)
 	while (inptr < inend && *inptr != '\n') {
 		if (*inptr == ' ')
 			inptr++;
-		if (inptr >= inend || *inptr != '>') {
+		if (inptr >= inend || *inptr != '>')
```

Comment:
If you make this change, then inptr++ should be on the same line. (see heartbleed bug)